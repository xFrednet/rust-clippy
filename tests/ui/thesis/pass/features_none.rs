//@rustc-env: CLIPPY_PETS_PRINT=1

#[warn(clippy::borrow_pats)]
fn main() {}
