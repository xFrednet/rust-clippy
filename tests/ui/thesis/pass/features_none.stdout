# "main"
- Body: Output(Unit)     , NotConst , Safe  , Sync , Free       {NoArguments}

